plugin/api.go

```diff
@@ -480,6 +480,36 @@ type API interface {
 	//
 	// Minimum server version: 5.7
 	SendMail(to, subject, htmlBody string) *model.AppError
+
+	// CreateBot creates the given bot and corresponding user.
```

Comment:
We talked about a way when designing this for plugins to avoid the trouble of creating and managing a bot themselves. Is there another ticket for that functionality? Has there been any more discussion on how that might operate?

Reply:
Are you talking about the idea of having a set of bots defined in the manifest?

Reply:
To my knowledge, no ticket exists as such, and no discussion on my end about how we could make that easier. I feel like this might be a job for that "external" plugin library that streamlines things (vs. building it into the package).

Reply:
Something like that, yes. With maybe a simple `GetMyBot` that can be called by the plugin?